.github/actions/generate-attestations/__tests__/attestation.test.ts

```diff
@@ -0,0 +1,77 @@
+const fs = require("fs");
+const statement = require("../src/attestation");
```

Comment:
since you added unit tests, maybe we can add these as pre-submits.

Reply:
done! updated contirbuting docs to mention jest (node.js test framework) and added to presubmits and Makefile